tests/containers/helm.pm

```diff
@@ -44,18 +44,21 @@ sub run {
     else {
         install_kubectl();
 
+        my $base = get_required_var("PUBLIC_CLOUD_CREDENTIALS_URL_BASE");
```

Comment:
It don't think it makes sense to create yet another variable. I think that `PUBLIC_CLOUD_CREDENTIALS_URL` is enough.

Reply:
yes, I agree. changed

Reply:
There are still remains…